packages/presentation/lib/utils/widget_display_helper.dart

```diff
@@ -0,0 +1,72 @@
+import 'package:domain/domain.dart';
+import 'package:flutter/cupertino.dart';
+import 'package:presentation/const/app.dart';
+import 'package:presentation/style/dimens.dart';
+
+enum ScreenType { desktop, tablet, phone }
+
+class WidgetDisplayHelper {
```

Comment:
I think it's better to split this file into smaller parts. it will grow too much if we'd have 10-20 screens and since most of the methods relate to the specific screen and is not used anywhere else, u can put it to the class level/into some screen utils

Reply:
I left WidgetDisplayHelper as presentation lvl utility class and create two separate classes for home and details which also use WidgetDisplayHelper.
and I left isBottomNavBarActive method in WidgetDisplayHelper, cuz this method used on Material app as equal presentation lvl